ovos_utils/log_parser.py

```diff
@@ -0,0 +1,506 @@
+import re
+import os
+from datetime import datetime
+from traceback import FrameSummary
+from dataclasses import dataclass
+from typing import Any, Tuple, List, Generator, Dict, Union, Optional
+
+from dateutil.parser import parse
+import rich_click as click
+from rich.console import Console
+from rich.prompt import Prompt
+from rich.style import Style
+from rich.table import Table
+import pydoc
+
+from ovos_config import Configuration
```

Comment:
i suppose here is fine to require ovos_config since it already has other extras, but since its only used for the time format we should also treat is as optional


speaking of which, needs extras requirements in setup.py for rich_click